src/casual.rs

```diff
@@ -29,7 +29,8 @@ impl Extractor for Casual {
 
     fn render(&self, out: &mut io::Write) -> Result<(), Error> {
         for entry in &self.entries {
-            write!(out, "{}\n", entry)?
+            // Add a horizontal line between the entries.
+            write!(out, "{}\n\n***\n\n", entry)?
```

Comment:
Why after an entry?

Reply:
Because it seems that each entry is a comment:

```rust
    fn extract(&mut self, comment: &str) -> bool {
        let url_pattern = Regex::new(r"(((http://)|(https://)|(ftp://)|(www\.))([a-zA-Z0-9_\-]+\.)*[a-zA-Z0-9\-]+\.[a-zA-Z0-9\-]+)").unwrap();
        let res = url_pattern.is_match(comment);
        if res {
            self.entries.push(comment.to_owned())
        }
        res
    }
```

Reply:
Did I misunderstand the code?

Reply:
https://github.com/dyweb/weekly/blame/master/2018/2018-04-30-weekly.md#L142

The line at the end of the post is weird.

Reply:
We :heart: 快糙猛